doc/nrf/config_and_build/configuring_app/cmake/index.rst

```diff
@@ -44,24 +44,301 @@ See :ref:`zephyr:zephyr-app-cmakelists` in the Zephyr documentation for more inf
+
+        * :guilabel:`Board`: ``nrf5340dk_nrf5340_cpuapp``
+        * :guilabel:`Extra CMake arguments`: ``-- -DEXTRA_CONF_FILE=nrf5340dk_app_sr_net.conf -Dmcuboot_EXTRA_CONF_FILE=nrf5340dk_mcuboot_sr_net.conf``
+
+     .. group-tab:: Command line
+
+        .. code-block::
+
+           west build -b nrf5340dk_nrf5340_cpuapp -- -DEXTRA_CONF_FILE=nrf5340dk_app_sr_net.conf -Dmcuboot_EXTRA_CONF_FILE=nrf5340dk_mcuboot_sr_net.conf
+
+Examples for EXTRA_DTC_OVERLAY_FILE
+-----------------------------------
+
+Providing one devicetree overlay file
+  The following example shows how you can provide an additional devicetree overlay file (:file:`usb.overlay`) when building for a build target (``nrf52840dk_nrf52840``):
+
+  .. tabs::
+
+     .. group-tab:: nRF Connect for VS Code
+
+        Use the following parameters when `setting up a build configuration <How to build an application_>`_:
+
+        * :guilabel:`Board`: ``nrf52840dk_nrf52840``
+        * :guilabel:`Extra CMake arguments`: ``-- -DEXTRA_DTC_OVERLAY_FILE=no-dfe.overlay``
+
+     .. group-tab:: Command line
+
+        .. code-block::
+
+           west build -b nrf52840dk_nrf52840 -- -DEXTRA_DTC_OVERLAY_FILE=no-dfe.overlay
+
+Providing devicetree overlay and Kconfig fragment files
+  The following example shows how you can provide an additional devicetree overlay file (:file:`usb.overlay`) and an additional Kconfig fragment file (:file:`prj_cdc.conf`) when building for a build target (``nrf52840dk_nrf52840``):
+
+  .. tabs::
+
+     .. group-tab:: nRF Connect for VS Code
+
+        Use the following parameters when `setting up a build configuration <How to build an application_>`_:
+
+        * :guilabel:`Board`: ``nrf52840dk_nrf52840``
+        * :guilabel:`Extra CMake arguments`: ``-- -DEXTRA_DTC_OVERLAY_FILE=usb.overlay -DEXTRA_CONF_FILE=prj_cdc.conf``
+
+     .. group-tab:: Command line
+
+        .. code-block::
+
+           west build -b nrf52840dk_nrf52840 -- -DEXTRA_DTC_OVERLAY_FILE=usb.overlay -DEXTRA_CONF_FILE=prj_cdc.conf
+
+Examples for SHIELD
+-------------------
+
+The following example shows how you can add shield support (``nrf7002eb``) when building for a build target (``thingy53_nrf5340_cpuapp``):
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      Use the following parameters when `setting up a build configuration <How to build an application_>`_:
+
+      * :guilabel:`Board`: ``thingy53_nrf5340_cpuapp``
+      * :guilabel:`Extra CMake arguments`: ``-- -DSHIELD=nrf7002eb``
+
+   .. group-tab:: Command line
+
+      .. code-block::
+
+         west build -b thingy53_nrf5340_cpuapp -- -DSHIELD=nrf7002eb
+
+Examples for CONF_FILE
+----------------------
+
+Select the build type
+  The following example shows how you can select a build type (here, ``prj_usb.conf`` of the :ref:`Zigbee NCP sample <zigbee_ncp_sample>`) when building for a build target (``nrf52840dk_nrf52840``):
```

Comment:
```suggestion
  The following example shows how you can select a build type (here, using the :file:`prj_usb.conf` configuration file of the :ref:`Zigbee NCP <zigbee_ncp_sample>` sample) when building for a build target (``nrf52840dk_nrf52840``):
```
`prj_usb.conf` is not a build type itself, it is the configuration file that is used to select the build type.

The example introductions are inconsistent on whether they include "here," when introducing what the sample command uses. I would recommend unifying one way or another. Alternatively, the introductory sentence could be split into two for all of them, along the following lines:
```suggestion
  The following example shows how you can select a build type when building for a build target.
  It uses the :file:`prj_usb.conf`` configuration file of the :ref:`Zigbee NCP <zigbee_ncp_sample>` sample to select the build type, and ``nrf52840dk_nrf52840`` as the build target.
```